docs/widgets.Rmd

```diff
@@ -99,7 +99,41 @@ to allow for interaction.  See the `?rgl::rglShared`
 help topic for details of `crosstalk` usage.
 
 ```r
-devtools::install_github("r-forge/rgl")
+devtools::install_github("dmurdoch/rgl")
+```
+
+
+---
+[![timevis screenshot](thumbnails/daattali-timevis.png){.widget-thumbnail}](https://github.com/daattali/timevis)
+
+### [timevis](https://github.com/daattali/timevis)
+
+**Author:** [Dean Attali](https://github.com/daattali)
+
+`timevis` lets you create interactive timeline visualizations in R.
+This package is based on the [visjs](https://visjs.github.io/vis-timeline/docs/timeline/)
+Timeline JavaScript library. Note that as of late 2024, the version of `timevis`
+with `crosstalk` support has not yet been released to CRAN.
```

Comment:
```suggestion
Timeline JavaScript library.
```

I think this warning is covered by using the `install_github()` command below. I'd also have more hope that timevis will be sent to CRAN before this page is updated again.